File: practica 1/sesion 3/DS_S3_C++Filters/ParteTecnico.cpp

#include "ParteTecnico.h"

ParteTecnico::ParteTecnico(){}

ParteTecnico::ParteTecnico(int amable, int profesional, int eficaz){
	setAmabilidad(amable);
	setProfesionalidad(profesional);
	setEficacia(eficaz);
}

int ParteTecnico::getAmabilidad() {
	return this->amabilidad;
}

void ParteTecnico::setAmabilidad(int unAmabilidad) {
	this->amabilidad = unAmabilidad;
}

int ParteTecnico::getProfesionalidad() {
	return this->profesionalidad;
}

void ParteTecnico::setProfesionalidad(int unProfesionalidad) {
	this->profesionalidad = unProfesionalidad;
}

int ParteTecnico::getEficacia() {
	return this->eficacia;
}

void ParteTecnico::setEficacia(int unEficacia) {
	this->eficacia = unEficacia;
}

File: practica 2/implementacion dart/lib/ModeloSOSEMADO/ParteTecnico.dart
import 'Trabajo.dart';

class ParteTecnico{
  int amabilidad, profesionalidad, eficacia;

  ParteTecnico(amable,profesional,eficaz){
      setAmabilidad(amable);
      setEficacia(eficaz);
      setProfesionalidad(profesional);
  }

  int getAmabilidad(){
    return this.amabilidad;
  }

  int getProfesionalidad(){
    return this.profesionalidad;
  }

  int getEficacia(){
    return this.eficacia;
  }

  void setAmabilidad(int amable){
    this.amabilidad = amable;
  }

  void setProfesionalidad(int profesional){
    this.profesionalidad = profesional;
  }

  void setEficacia(int eficaz){
    this.eficacia = eficaz;
  }
}